src/Aspire.Cli/DotNetCliRunner.cs

```diff
@@ -468,6 +468,22 @@ private async Task StartBackchannelAsync(Process process, string socketPath, Tas
                     // We don't want to spam the logs with our early connection attempts.
                 }
             }
+            catch (AppHostIncompatibleException ex)
+            {
+                logger.LogError(
+                    ex,
+                    "AppHost is incompatible with the CLI. The AppHost must be updated to a version that supports the {RequiredCapability} capability.",
```

Comment:
Do we use this casing everywhere and does this show up by default?
```suggestion
                    "The app host is incompatible with the CLI and must be updated to a version that supports the {RequiredCapability} capability.",
```